content/doc/developer/tutorial-improve/update-parent-pom.adoc

```diff
@@ -19,6 +19,52 @@ The most recent parent POM files also provide additional static analysis like sp
 // Create the branch
 include::doc/developer/tutorial-improve/_create-a-branch.adoc[]
 
+
+== Maven configuration
+
+Before updating the parent POM (Project Object Model) using the "mvn -ntp version:update-parent" command, you must make sure that the Maven configuration files are correctly configured.
+This is important because the plugin:maven-hpi-plugin[maven-hpi-plugin], used for developing Jenkins plugins, might require some additional global configurations to function correctly with older plugin environments.
+
+If Maven reports that it cannot download artifacts during the update process, it might be due to improper configuration settings.
```

Comment:
```suggestion
Improper configuration settings can cause Maven to report that it cannot download artifacts during the update process.
```
Suggestion to reword the sentence and use active voice.

Reply:
yes sir done that